pkg/apis/core/validation/seed.go

```diff
@@ -90,6 +86,15 @@ func ValidateSeedSpec(seedSpec *core.SeedSpec, fldPath *field.Path, inTemplate b
 		allErrs = append(allErrs, field.Required(providerPath.Child("region"), "must provide a provider region"))
 	}
 
+	zones := sets.NewString()
+	for i, zone := range seedSpec.Provider.Zones {
+		if zones.Has(zone) {
```

Comment:
Duplicate check is the only validation done. Do you think that zone names should also be validated else an operator can make a mistake in giving an incorrect name for a zone and with no validation it will never complain?

Reply:
I don't think we can perform any further more sophisticated validation here. It's like the `region`. The `Seed` is just any cluster we don't know anything about. For `ManagedSeed`s, we already have validation to ensure the zones in the `Shoot` spec are present in the `zones` in the `Seed` spec.

Reply:
Yeah, especially for non-hyperscalers this might not be true.